### PR TITLE
Conor/library

### PR DESCRIPTION
**Pull request by c-brenn**
This is p much ready to go. Might be best to wait till we have a list of books ready to upload?

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -11,7 +11,17 @@
...
+
+  create_table "books", force: :cascade do |t|
+    t.string   "title"
+    t.string   "author"
+    t.string   "synopsis"
+    t.string   "image_url"
+    t.boolean  "available"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
```
Comment (houli) [inline db/schema.rb:24]: Would like to add a field which shows the person who took out the book. Would only be for the admin interface and for when someone is taking out a book with the app.

Comment (houli) [inline db/schema.rb:24]: Actually also an ISBN column. That will be used to lookup the scanned book.

**Comment by c-brenn** [comment]
Definitely. It, could set up a little cronjob to send off reminder emails too

File: db/schema.rb (hunk 1 above)
Comment (houli) [inline db/schema.rb:24]: More model stuff but there can be more book states. Can be modeled with AR enum. https://github.com/DUCSS/ducss-site-old/blob/master/project/library/models.py shows some of the model choices made for the old version

**Comment by PinPinIre** [comment]
This PR dead?

**Comment by CalvinNolan** [comment]
Yeah #39 this will come before the library

**Comment by c-brenn** [comment]
R I P

**Pull request closed without merging**